gwpy/timeseries/timeseries.py

```diff
@@ -1749,11 +1749,122 @@ def notch(self, frequency, type='iir', filtfilt=True, **kwargs):
                                   type=type, **kwargs)
         return self.filter(*zpk, filtfilt=filtfilt)
 
+    def q_gram(self, qrange=(4, 64), frange=(0, numpy.inf), gps=None,
```

Comment:
What purpose does this method serve in the `TimeSeries` object? I think it might be better exposed simply via the `gwpy.signal.qtransform` module for expert users (like yourself) to use since the returned object doesn't really have much utility, except as a stepping-stone towards an `EventTable` or a `Spectrogram`.